File: src/main/java/name/hennr/series/stalker/tvmaze/Episode.java
package name.hennr.series.stalker.tvmaze;

public class Episode {

    public int id;
    public String airdate;

    // many fields are omitted here

    @Override
    public String toString() {
        return "Episode{" +
                "id=" + id +
                ", airdate='" + airdate + '\'' +
                '}';
    }
}

File: src/main/java/name/hennr/series/stalker/tvmaze/Episode.kt
package name.hennr.series.stalker.tvmaze

class Episode {
    var id = 0
    @kotlin.jvm.JvmField
    var airdate: String? = null

    // many fields are omitted here
    override fun toString(): String {
        return "Episode{" +
                "id=" + id +
                ", airdate='" + airdate + '\'' +
                '}'
    }
}
